### ISSUE TITLE
feathers-hooks-common/lib/populate, feathers-hooks-common/lib/bundled not found.

### ISSUE THREAD

**Issue by bdschr** [feathersjs/feathers-hooks#149]
Follow up:
https://github.com/feathersjs/feathers-hooks/issues/142

```
These dependencies were not found:

* feathers-hooks-common/lib/populate in ./~/feathers-hooks/lib/hooks.js
* feathers-hooks-common/lib/bundled in ./~/feathers-hooks/lib/hooks.js

To install them, you can run: npm install --save feathers-hooks-common/lib/populate feathers-hooks-common/lib/bundled

ERROR in ./~/feathers-hooks/lib/hooks.js
Module not found: Error: Can't resolve 'feathers-hooks-common/lib/populate' in '/Users/david/Data/git/node/test/node_modules/feathers-hooks/lib'
 @ ./~/feathers-hooks/lib/hooks.js 13:16-61
 @ ./~/babel-loader/lib?{"presets":["vue-app"],"cacheDirectory":true}!./~/vue-loader/lib/selector.js?type=script&index=0!./pages/auth/login.vue
 @ ./pages/auth/login.vue
 @ ./.nuxt/router.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?reload=true ./.nuxt/client.js
ERROR in ./~/feathers-hooks/lib/hooks.js
Module not found: Error: Can't resolve 'feathers-hooks-common/lib/bundled' in '/Users/david/Data/git/node/test/node_modules/feathers-hooks/lib'
 @ ./~/feathers-hooks/lib/hooks.js 15:15-59
 @ ./~/babel-loader/lib?{"presets":["vue-app"],"cacheDirectory":true}!./~/vue-loader/lib/selector.js?type=script&index=0!./pages/auth/login.vue
 @ ./pages/auth/login.vue
 @ ./.nuxt/router.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?reload=true ./.nuxt/client.js
```

**Comment by eddyystop** [comment]
@daffl in these lines https://github.com/feathersjs/feathers-hooks/blob/master/src/hooks.js#L3-L7

either require directly from `feathers-hooks-common`. (bundled.js for example, no longer exists.)

**Comment by daffl** [comment]
Yes but the current version of feathers-hooks is still using v2.0. I think I'm going to make a feathers-hooks v2.0 with the bundled hooks removed.

**Comment by daffl** [comment]
What does `npm ls feathers-hooks-common` show?

**Comment by bdschr** [comment]
test@0.0.0 /Users/david/Data/git/node/test
├─┬ feathers-hooks@1.8.1
│ └── feathers-hooks-common@2.0.3 
└── feathers-hooks-common@3.0.0

**Comment by iFarmGolems** [comment]
Same issue here.

**Comment by daffl** [comment]
The issue is probably that Webpack does not seem to know which `feathers-hooks-common` to bundle. `feathers-hooks` is using v2.x and it looks like you can not also install v3.x

**Comment by daffl** [comment]
This is solved now by upgrading to v2.0.0 which removed the bundled hooks (and dependency on the old `feathers-hooks-common`). 

For the bundled hooks use [feathers-hooks-common](https://docs.feathersjs.com/api/hooks-common.html) can be used.
